Content.Server/Atmos/AtmosDirection.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Robust.Shared.Maths;
+
+namespace Content.Server.Atmos
```

Comment:
This should be moved to Content.Shared, as the Atmospherics constants refers to it through a doc comment.